sync-todo/v2/client/maui/RealmTodo/realm.json

```diff
@@ -0,0 +1,4 @@
+﻿{
+  "AppId": "todo-template-tldrx",
```

Comment:
Sorry, due to the way the backend works, this must have the casing: `appId` and `baseUrl`.

Reply:
Fixed